src/Sema.zig

```diff
@@ -23266,7 +23277,14 @@ fn validateExternType(
         .Float,
         .AnyFrame,
         => return true,
-        .Pointer => return !ty.isSlice(),
+        .Pointer => {
+            const pointee_ty = ty.elemType2();
+            if (pointee_ty.zigTypeTag() == .Pointer) {
```

Comment:
This branch shouldn't be necessary as `typeRequiresComptime` already does it.

Reply:
But what if we have pointer to a slice?

Reply:
A pointer to a slice is a valid extern type as long as the slice isn't pointing to a comptime only type.